[opencascade] cmake failure

**Host Environment**
- OS: Windows
- Compiler: Visual Studio 16 2019
- CMake: 3.18.4

**To Reproduce**
Steps to reproduce the behavior:
`vcpkg\vcpkg.exe install opencascade:x64-windows`

**CMakeLists.txt**
```cmake
cmake_minimum_required(VERSION 3.5)

project(app LANGUAGES CXX)

set(CMAKE_INCLUDE_CURRENT_DIR ON)

set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)

find_package(OpenCASCADE REQUIRED)
```

**cmake build commands**
```
mkdir build
cd build
cmake .. -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake
```

**Failure logs** 
```
-- Building for: Visual Studio 16 2019
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.17763.
-- The CXX compiler identification is MSVC 19.27.29112.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/BuildTools/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at C:/vcpkg/installed/x64-windows/share/opencascade/OpenCASCADEFoundationClassesTargets.cmake:84
(message):
  The imported target "TKernel" references the file

     "C:/vcpkg/installed/x64-windows/debug/libd/TKernel.lib"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "C:/vcpkg/installed/x64-windows/share/opencascade/OpenCASCADEFoundationClassesTargets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  C:/vcpkg/installed/x64-windows/share/opencascade/OpenCASCADEConfig.cmake:95 (include)
  C:/vcpkg/scripts/buildsystems/vcpkg.cmake:555 (_find_package)
  CMakeLists.txt:16 (find_package)

```

**Additional context**
It seems to be looking for debug version, so I have tried `-DCMAKE_BUILD_TYPE=Release` with the same result.
I will start looking in the `OpenCASCADEFoundationClassesTargets.cmake` and `OpenCASCADEConfig.cmake` files for more clues

--- a/ports/opencascade/CONTROL
+++ b/ports/opencascade/CONTROL
@@ -1,5 +1,6 @@
 Source: opencascade
 Version: 7.4.0
+Port-Version: 1
 Build-Depends: freetype
 Description: Open CASCADE Technology (OCCT) is an open-source software development platform for 3D CAD, CAM, CAE.
 Supports: !(uwp|osx|linux|arm|static)

--- a/ports/opencascade/portfile.cmake
+++ b/ports/opencascade/portfile.cmake
@@ -50,6 +50,23 @@ file(RENAME ${CURRENT_PACKAGES_DIR}/debug/libd ${CURRENT_PACKAGES_DIR}/debug/lib
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 
+# fix paths in target files
+list(APPEND TARGET_FILES 
+    ${CURRENT_PACKAGES_DIR}/share/opencascade/OpenCASCADEApplicationFrameworkTargets-debug.cmake
+    ${CURRENT_PACKAGES_DIR}/share/opencascade/OpenCASCADECompileDefinitionsAndFlags-debug.cmake
+    ${CURRENT_PACKAGES_DIR}/share/opencascade/OpenCASCADEDataExchangeTargets-debug.cmake
+    ${CURRENT_PACKAGES_DIR}/share/opencascade/OpenCASCADEFoundationClassesTargets-debug.cmake
+    ${CURRENT_PACKAGES_DIR}/share/opencascade/OpenCASCADEModelingAlgorithmsTargets-debug.cmake
+    ${CURRENT_PACKAGES_DIR}/share/opencascade/OpenCASCADEModelingDataTargets-debug.cmake
+    ${CURRENT_PACKAGES_DIR}/share/opencascade/OpenCASCADEVisualizationTargets-debug.cmake
+)
+foreach(TARGET_FILE ${TARGET_FILES})
+    file(READ ${TARGET_FILE} filedata)
+    string(REGEX REPLACE "libd" "lib" filedata "${filedata}")
+    string(REGEX REPLACE "bind" "bin" filedata "${filedata}")
+    file(WRITE ${TARGET_FILE} ${filedata})
+endforeach()
+
 # the bin directory ends up with bat files that are noise, let's clean that up
 file(GLOB BATS ${CURRENT_PACKAGES_DIR}/bin/*.bat)
 file(REMOVE_RECURSE ${BATS})

